### DISCUSSION TITLE
CDN issue

### DISCUSSION THREAD

**Discussion by hrstoyanov** [gnat/surreal#37] [Ideas]
How can I download a particular version?

for example version 1.2.1 does not download from this:
```https://cdn.jsdelivr.net/gh/gnat/surreal@1.2.1/surreal.js```

Also, can you, please, offer a minified version (no comments inside,  at least)?

Also, can you, please, do the same for you other awesome project [css-scope-inline](https://github.com/gnat/css-scope-inline)

**Comment by gnat** [comment]
People are welcome to do as they please in their own workflows, but minified / obfuscated releases are not currently planned because promoting "builds" unfortunately tends to put major pressure on all developers to use them.

This sucks because we lose the debuggable, auditable, readable nature of surreal and it leads to more eventual complexity: source maps, dependencies, and other DX friction. The benefits will be inconsequential for most, particularly if the server has gzip or zstd enabled.
